python/sanlock.c

```diff
@@ -1693,12 +1691,10 @@ __module_init(PyObject* m)
         return -1;
     }
 
-#define PYSNLK_INIT_ADD_CONSTANT(x, y) \
-    if ((sk_constant = PyInt_FromLong(x)) != NULL) { \
-        if (PyModule_AddObject(m, y, sk_constant)) { \
-            Py_DECREF(sk_constant); \
-            return -1; \
-        } \
+#define PYSNLK_INIT_ADD_CONSTANT(value, label) \
+    if (PyModule_AddIntConstant(m, label, value) != 0) {\
+        Py_DECREF(py_exception); \
+        return -1; \
```

Comment:
Please don't add macro with return. This is very confusing to use. Flow should never be hidden in a macro.

Reply:
okay, the rest of the function code has the form
AddConstXXX(..)
AddConstXXX(..)
AddConstXXX(..)
etc.
dont we want to check if some const add failed and fail the entire function in this case?
it could save typing multiple ifs.